File: src/Services/WareHouse/WareHouse.Domain/Entity/BeginningWareHouse.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class BeginningWareHouse:BaseEntity
    {
        public string WareHouseId { get; set; }
        public string ItemId { get; set; }
        public string UnitId { get; set; }
        public string UnitName { get; set; }
        public int Quantity { get; set; }
        public DateTime CreatedDate { get; set; }
        public string CreatedBy { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string ModifiedBy { get; set; }

        public virtual WareHouseItem Item { get; set; }
        public virtual Unit Unit { get; set; }
        public virtual WareHouse WareHouse { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/entity/BeginningWareHouse.ts
import { WareHouseItem } from './WareHouseItem';
import { Inward } from './Inward';
import { BaseEntity } from './BaseEntity';
import { Unit } from './Unit';
import { WareHouse } from './WareHouse';
export interface BeginningWareHouse extends BaseEntity {
    wareHouseId: string;
    itemId: string;
    unitId: string;
    unitName: string;
    quantity: number;
    createdDate: string;
    createdBy: string;
    modifiedDate: string;
    modifiedBy: string;
    item: WareHouseItem;
    unit: Unit;
    wareHouse: WareHouse;
}